src/homepage/_posts/2021-03-16-james-3.6.0.markdown

```diff
@@ -0,0 +1,165 @@
+- JAMES-3405 Expose metrics of Guice servers over HTTP - enables easy Prometheus metrics collection
+- JAMES-3407 Distributed server: Read-repairs for the mailbox entity
+- JAMES-3428 Distributed server: Read-repairs for the mailbox counters entity
+- JAMES-3139 Expose RabbitMQ channel & connection configuration
+- JAMES-3441 Make possible and document Distributed Server setup with specialized instances
+- JAMES-3337 Document the use of JWT
+- JAMES-3399 Allow JSON logging with logback - enables structure logging with FluentBit
+- JAMES-3396 WebAdmin should try to prevent RRT addresses redirection loops when possible
+- JAMES-3402 JMAP MDN messages should have a Date header
+- JAMES-3028 Distributed server: allow choosing whether blobs should be deduplicated
+- JAMES-3196 CanSendFromImpl: enable to send email from aliases for SMTP and JMAP
+- JAMES-3196 Add an IMAP SessionId to correlate logs
+- JAMES-3502 DistributedServer: SSL and authentication support for RabbitMQ
+- JAMES-3504 Metrics and log for POP3
+- JAMES-3431 Optional DSN support
+- JAMES-3202 Allow search index Reindexing without cleanup
+
+### Changed
+- Switch to Java 11 for build and run
+- JAMES-2760 mailqueue.size.metricsEnabled should be false by default
+- JAMES-3252 DomainList autoDetection should be turned off by default. Operators relying on implicit values for enabling DomainList autoDetection now needs to explicitly configure it.
+- JAMES-3184 Throttling mechanism allow an admin to specify the throughput desired for a given WebAdmin task
+- JAMES-3224 Configuration for Cassandra ConsistencyLevel.{QUORUM, SERIAL} (for multi-dc configuration)
+- JAMES-3176 Rewritte MDN parsing with Parboiled scala (avoid asm library dependency clash within the Distributed Server)
+- JAMES-3194 Rely on DTOConverter in TaskRoute
+- JAMES-3430 Restructure message properties storage within Cassandra Mailbox. See upgrade instructions.
+- JAMES-3435 Use EventSourcing to manage ACL - avoid SERIAL reads for ACL thus unlocking a performance enhancement for the Distributed James server. Read upgrade instructions.
+- JAMES-2124 Sorts module declarations in reactors (thanks to Jean Helou)
+- JAMES-3440 JMAP users can now avoid relying on ElasticSearch reads for basic listing operations thanks to the EmailQueryView 
+- JAMES-3252 DomainList autoDection should be turned off by 
+- JAMES-3192 Upgrade Apache configuration to 2.7
+- JAMES-3492 Upgrade ElasticSearch dependency for DistributedServer to 7.10
+- JAMES-2514 Upgrade Cassandra dependency for DistributedServer 3.11.3 -> 3.11.10
+- JAMES-3497 Multiple dependencies upgrades
+- JAMES-3499 Package LDAP in Distributed Server
+- JAMES-3225 Set up of the Apache CI
+- [REFACTORING] Switch most of the test suite to JUNIT 5
+
+### Deprecated
+- HybridBlobStore. This will be removed after 3.6.0 release. Introduced to fasten small blob access, its usage could be
```

Comment:
I don't use it but will it be removed in 3.6.1 or in 3.7.0 ? ( i assume the latter but ... )